test/run_pass/test_os.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var assert = require('assert');
+var common = require('../common');
+var os = require('os');
+
+var hostname = os.hostname();
+common.is.string(hostname);
+assert.ok(hostname.length > 0);
+
+var uptime = os.uptime();
+common.is.number(uptime);
+assert.ok(uptime > 0);
+
+var release = os.release();
+common.is.string(release);
+assert.ok(release.length > 0);
+
+var platform = os.platform();
+common.is.string(platform);
+assert.ok(platform.length > 0);
+
+var EOL = os.EOL;
+if (common.isWindows) {
+  assert.strictEqual(EOL, '\r\n');
+} else {
+  assert.strictEqual(EOL, '\n');
+}
```

Comment:
We do not need the above tests at this PR, could you please remove them?

Reply:
I will do!